docs/the_black_code_style.md

```diff
@@ -289,6 +289,16 @@ If you are adopting _Black_ in a large project with pre-existing string conventi
 you can pass `--skip-string-normalization` on the command line. This is meant as an
 adoption helper, avoid using this for new projects.
 
+As an experimental option, _Black_ splits long strings (using parentheses where
+appropriate) and merges short ones. When split, parts of f-strings that don't need
+formatting are converted to plain strings. User-made splits are respected when they do
+not exceed the line length limit. Line continuation backslashes are converted into
+parenthesized strings. Unnecessary parentheses are stripped. To enable experimental
+string processing, pass `--experimental-string-processing` on the command line.
```

Comment:
It might be a good idea to explicitly call out that feedback (and bug reports too!) for this feature is welcomed and even encouraged. The experimental nature of it already sends that message, but IMO it can't hurt to make it explicit.

Reply:
Yeah that's a good idea!